pages/favs.jsx

```diff
@@ -3,18 +3,64 @@ import { getFavs } from '../js/favs-manager';
 import ProjectList from '../components/project-list/project-list.jsx';
 import HintMessage from '../components/hint-message/hint-message.jsx';
 
+import Service from '../js/service.js';
+import config from '../config/config.js';
+import env from "../config/env.generated.json";
+
 export default React.createClass({
   getInitialState() {
     return {
-      favs: null
+      favs: null,
+      loggedInCheckDateTime: null
     };
   },
+  verifyLoggedInStatus() {
+    Service.nonce()
+      .then((response) => {
+        this.updateLoggedInInfo(true, response.user);
+      })
+      .catch((reason) => {
+        this.updateLoggedInInfo(false);
+        console.error(reason);
+      });
+  },
+  updateLoggedInInfo(status, username = ``) {
+    config.userLoggedInStatus.set(status);
+    config.username.set(username);
+    this.setState({loggedInCheckDateTime: new Date()});
```

Comment:
`new Date()` is a Date object, which is almost certainly not what you want (for a string, you want to call `(new Date()).toLocaleString()` or the like, for a timestamp you want `Date.now()`), but more importantly: this state variable isn't used anywhere (not passed down, not used in rendering, etc) so let's remove it?